src/app/clusters/ecosystem-information-server/ecosystem-information-server.cpp

```diff
@@ -217,7 +217,7 @@ std::unique_ptr<EcosystemLocationStruct> EcosystemLocationStruct::Builder::Build
     VerifyOrReturnValue(!mIsAlreadyBuilt, nullptr, ChipLogError(Zcl, "Build() already called"));
     VerifyOrReturnValue(!mLocationDescriptor.mLocationName.empty(), nullptr, ChipLogError(Zcl, "Must Provided Location Name"));
     VerifyOrReturnValue(mLocationDescriptor.mLocationName.size() <= kLocationDescriptorNameMaxSize, nullptr,
-                        ChipLogError(Zcl, "Must Location Name must be less than 64 bytes"));
+                        ChipLogError(Zcl, "Must Location Name must be less than 128 bytes"));
```

Comment:
why not `%u` and use kLocationDescriptorNameMaxSize instead of hardcoding the size.